selenium_tests/CanonizerAccountSettingsPage.py

```diff
@@ -390,32 +420,50 @@ def verify_phone_number_with_blank_phone_number(self):
     def verify_phone_number_with_invalid_length_phone_number(self):
         self.find_element(*AccountSettingsManageProfileInfoIdentifiers.PHONE_NUMBER).clear()
         self.verify_phone_number('1234')
-        return self.find_element(*AccountSettingsManageProfileInfoIdentifiers.ERROR_PHONE_NUMBER).text
+        error = self.find_element(*AccountSettingsManageProfileInfoIdentifiers.ERROR_PHONE_NUMBER).text
+        if error == 'The phone number must be 10 digits.':
+            return AccountSettingsMySupportsPage(self.driver)
+
+    def verify_phone_number_with_characters(self):
+        self.find_element(*AccountSettingsManageProfileInfoIdentifiers.PHONE_NUMBER).clear()
+        self.verify_phone_number('233cc')
+        error = self.find_element(*AccountSettingsManageProfileInfoIdentifiers.ERROR_PHONE_NUMBER).text
+        if error == 'The phone number must be 10 digits.':
+            return AccountSettingsMySupportsPage(self.driver)
 
-    def update_profile_with_invalid_first_name(self, firstname, middle_name, last_name, email, language, dob, address_line1, address_line2, city, state, zip_code):
+    def update_profile_with_invalid_first_name(self, firstname, middle_name, last_name, email, language, dob,
+                                               address_line1, address_line2, city, state, zip_code):
         self.find_element(*AccountSettingsManageProfileInfoIdentifiers.FIRST_NAME).clear()
-        self.update(firstname, middle_name, last_name, email, language, dob, address_line1, address_line2, city, state, zip_code)
-        return self.find_element(*AccountSettingsManageProfileInfoIdentifiers.ERROR_FIRST_NAME).text
+        self.update(firstname, middle_name, last_name, email, language, dob, address_line1, address_line2, city, state,
+                    zip_code)
+        error = self.find_element(*AccountSettingsManageProfileInfoIdentifiers.ERROR_FIRST_NAME).text
+        if error == 'The first name must be in alphabets and space only.':
+            return AccountSettingsMySupportsPage(self.driver)
 
-    def update_profile_with_invalid_middle_name(self, firstname, middle_name, last_name, email, language, dob, address_line1, address_line2, city, state, zip_code):
+    def update_profile_with_invalid_middle_name(self, firstname, middle_name, last_name, email, language, dob,
+                                                address_line1, address_line2, city, state, zip_code):
         self.find_element(*AccountSettingsManageProfileInfoIdentifiers.MIDDLE_NAME).clear()
-        self.update(firstname, middle_name, last_name, email, language, dob, address_line1, address_line2, city, state, zip_code)
-        return self.find_element(*AccountSettingsManageProfileInfoIdentifiers.ERROR_MIDDLE_NAME).text
+        self.update(firstname, middle_name, last_name, email, language, dob, address_line1, address_line2, city, state,
```

Comment:
Use array instead of long list of elements.

Reply:
Done